data_taking_scripts/motor_control.py

```diff
@@ -1,78 +1,54 @@
-from dripline.core import Interface
-import time
-import common_functions
+from motor import OrpheusMotors
+from data_logging import DataLogger
 #setting up connection to dripline
 auths_file = '/etc/rabbitmq-secret/authentications.json'
-the_interface = Interface(dripline_config={'auth-file': auths_file})
 
-def mm_to_inch(dist):
-    return dist/25.4
+def cm_to_inch(dist):
+    return dist/2.54
+def inch_to_cm(dist):
+    return dist*2.54
 
-#distance to move
-print("PLEASE ENTER DISTANCE IN MILLIMETERS (mm)")
-cavity_length_tracker = mm_to_inch(float(input('Enter initial resonator length (in mm): ')))
-distance_to_move = float(input('Enter the distance to move in mm (Empty cavity modemap is usually 30): '))
-resolution = float(input('Enter the number of measurements needed: '))
-increment_distance = distance_to_move/resolution
+# names should be exactly like 'curved_mirror', 'top_dielectric_plate', 'bottom_dielectric_plate'
+motors_to_move = ['curved_mirror', 'bottom_dielectric_plate', 'top_dielectric_plate']
+
+cavity_length_tracker = cm_to_inch(float(input('Enter initial resonator length (in cm): ')))
```

Comment:
change to resonator.
Also, this is not the resonator length. 
Make it obvious what distance you want. In this case, it's the distance between the inner faces of the mirror holders.